Sources/SwagGenKit/CodeFormatter.swift

```diff
@@ -310,15 +310,14 @@ public class CodeFormatter {
         // Responses
 
         let responses = operation.responses
-        let successResponse = responses.first { $0.successful }
         let successResponses = responses.filter { $0.successful }.map(getResponseContext)
         let failureResponses = responses.filter { !$0.successful }.map(getResponseContext)
 
-        context["responses"] = responses.map(getResponseContext)
+        context["responses"] = successResponses + failureResponses
         context["successResponse"] = successResponses.first
-        context["successType"] = successResponse.flatMap(getResponseContext)?["type"]
-        context["defaultResponse"] = responses.first { $0.statusCode == nil }.flatMap(getResponseContext)
-        context["onlySuccessResponses"] = successResponse != nil && responses.count == 1
```

Comment:
In this PR I'm keeping the current behavior.

I wonder:
shouldn't this be `successResponses.count == responses.count`?

Reply:
Yep, that would make sense. The behaviour would be slightly different (this would be true with 2 success responses, where before it would be false), but the way it's used in the template, it would actually be more correct

Reply:
Would you like me to make this change? 

I haven't dove much deeper into the library: I'm not sure what this change could cause.

Reply:
Yeah happy with it. It better reflects the name

Reply:
Done! I've also updated the CHANGELOG. I think this PR is ready to go! 🎉

Reply:
Actually this does seem to have broken something. You can see the result if you run `swift test` and see the generated diff

Reply:
I restored the previous behavior: now all tests pass!